server/src/main/java/org/elasticsearch/repositories/blobstore/BlobStoreRepository.java

```diff
@@ -912,12 +916,24 @@ public void cleanup(long repositoryStateId, Version repositoryMetaVersion, Actio
         ).collect(Collectors.toList());
     }
 
-    private List<String> cleanupStaleRootFiles(List<String> blobsToDelete) {
+    private List<String> cleanupStaleRootFiles(Collection<SnapshotId> deletedSnapshots, List<String> blobsToDelete) {
         if (blobsToDelete.isEmpty()) {
             return blobsToDelete;
         }
         try {
-            logger.info("[{}] Found stale root level blobs {}. Cleaning them up", metadata.name(), blobsToDelete);
+            if (logger.isInfoEnabled()) {
```

Comment:
I think it would be helpful to continue to log all the blobs at `TRACE` level; I'd do both if there are unexpected ones as it's interesting to know that they were unexpected.

Reply:
D'oh just spotted this was a backport and already approved in `master`, sorry 😕

Reply:
No worries, I'll do a follow-up for this shortly, I wanted to add some other trace logging as well anyway :)

Reply:
the more :eyes: the better

Reply:
https://github.com/elastic/elasticsearch/pull/56336 :)